scipy/stats/_stats_py.py

```diff
@@ -1046,14 +1046,14 @@ def moment(a, moment=1, axis=0, nan_policy='propagate'):
 
     Notes
     -----
-    The k-th central moment of a data sample is:
+    The k-th moment of a data sample is:
 
     .. math::
 
-        m_k = \frac{1}{n} \sum_{i = 1}^n (x_i - \bar{x})^k
```

Comment:
Something is going to be subtracted from the `x_i`. Chose a new symbol, e.g. `c`, and below define $c$ to be defined by `center`.